packages/influx-metrics-tracker/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import { Logger } from '@ovotech/winston-logger';
+import { InfluxDB } from 'influx';
+
+interface MetricsMeta {}
```

Comment:
Could this interface be removed in place of a Map or object type on the `staticMeta` parameter?